MAUI/Themes/Keys.md

```diff
@@ -107,3 +107,2093 @@ This page lists the keys for each control and the element to which it is mapped
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of a treemap item's border when it is selected.
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapItemHoverStroke      
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of a treemap item's border when it is hovered. 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapSelectedItemHoverStroke       
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of selected item hovered in treemap.  
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapToolTipNormalBackground        
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background color of tooltip in treemap. 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapToolTipNormalStroke         
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of tooltip stroke in treemap.  
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapToolTipNormalTextColor          
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of tooltip text in treemap.  
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapGroupItemNormalBackground           
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Background color of group item in treemap. 
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapGroupItemNormalStroke           
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of group item stroke.   
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapGroupItemNormalTextColor             
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of group item text.  
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapLegendNormalTextColor              
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of legend text in treemap.     
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapItemKeyboardFocusedStroke               
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Color of keyboard focused stroke in treemap.     
+            <br/>
+            <br/>
+        </td>
+    </tr>
+
+	<tr>
+    <td>
+            SfTreeMapSelectedItemStrokeWidth                
+            <br/>
+            <br/>
+        </td>
+        <td>
+            Size of selected item in treemap.      
```

Comment:
Size of selected item border in treemap.

Reply:
Updated